docs/setup/install/ingress-setup.md

```diff
@@ -14,7 +14,8 @@ You can use [this yaml file](https://github.com/devtron-labs/devtron/blob/main/m
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  annotations: nginx.ingress.kubernetes.io/app-root: /dashboard
+  annotations: 
+    nginx.ingress.kubernetes.io/app-root: /dashboard
```

Comment:
what is the difference between the previous `annotations` and the one you corrected?

Reply:
the previous one will create error regarding indentation in YAML file.
putting them in same line cause error.